### PR TITLE
Add lua script to perform OLTP r/w operations on table that has a bit…

### PR DESCRIPTION
**Pull request by Tusamarco**
… more complex structure and Varchar.

Add also a function in sb_rand to produce a pure alpha numeric string given existing was polluting the string with special char and so on.
Also a preliminary step to include chars that are of different language like arabic/chinese and so on

Still working on adding more complex Selects though.

@akopytov was wondering if could make sense to merge the code, especially for the varchar creation, or if you prefer to keep things separate. 
let me know

### CODE REVIEW

**Comment by akopytov** [comment]
Hi Marco,

This PR adds multiple things, I'm going to address the separately.

# 1. `sysbench.rand.varstringalpha()`


This can be implemented purely in Lua with something like this:

```lua
local location = sysbench.rand.string(string.rep('@', sysbench.rand.default(10, 100)))
```

But generally, I'm going to replace random strings API with something more flexible like the one described in https://github.com/akopytov/sysbench/issues/189

Because people have a rather broad range of different requirements to generated random strings, so I feel like I have to provide a more adequate tool for the job.

# 2. The `--stats-format` option. 

I like the idea and I think it should be a global option, rather than a test-specific one. I have reported https://github.com/akopytov/sysbench/issues/215 to implement it.

# 3. A new benchmark script.

It's good to know there are new benchmarks written in sysbench Lua, like this one, or TPCC. This is precisely why I'm putting so much effort into Lua -- I want other people to create benchmarks as easily as possible.

However, I'm reluctant to merge new benchmarks into the sysbench distribution. Let me explain.

Merging user Lua scripts into the main source tree makes sense from the user perspective -- it makes those scripts easily availably. But that also increases maintenance burden on me -- people will expect _me_ to document it, make sure it is compatible with future sysbench changes, fix any bugs they encounter, etc.

Just as an example, this PR currently fails CI tests. And Lua code does not always follow Lua best practices (like using global variables where local ones would be sufficient, etc.) So I would have to do some non-trivial work even to merge it, let alone support it in the future.

Since my resources are extremely limited, sysbench is just my hobby project after all, I'm aiming for the opposite direction -- make sysbench a small universal benchmark engine, and then move all benchmarks (including OLTP) to some external repositories. Also, I want to make sure those repositories are easily discoverable and usable.

Luckily, Lua already has all infrastructure to implement that, it requires relatively minor efforts on my end. It's called LuaRocks -- Lua's internal package manager. https://luarocks.org/

Writing LuaRock spec files is easy, see for example: https://github.com/luarocks/luarocks/wiki/Creating-a-rock

Basically, I just need to create my own LuaRocks repository with sysbench-specific benchmark scripts and other Lua packages that people find useful when implementing benchmarks (like Mongo connectors, or JSON and UUID libraries, for example)

The interface from the sysbench side should look like this:

```console
sysbench rocks search <name>
sysbench rocks install <name>
etc.
```

Again, this is all not too hard to implement, it's just one of a million things on my TODO.

For now, I'm happy for any third-party benchmark to exist in separate Github repositories.

Once the central repository for user-contributed script is available, I'm going to announce it through my blog and PlanetMySQL.

Thanks for using and improving sysbench!

**Comment by Tusamarco** [comment]
Hi @akopytov  I understand... just one note about the sysbench.rand.varstringalpha.
As mention I was adding that because I was focusing on having multiple language support and special UTF8 characters.
About using:
` local location = sysbench.rand.string(string.rep('@', sysbench.rand.default(10, 100)))`
I did use it an I had issues, because the function include characters like quote & double quote or "\" which create errors in mysql at the moment of the inserts.
Given that I choose to go ahead and have (in the meantime)  something base on basic characters.

**Comment by akopytov** [comment]
@Tusamarco hm, I don't understand how that code can result in non-letter characters. It generates a random sequence of characters in the ['a', 'z'] range, which only includes lowercase letters. Are you sure you were testing exactly the same code?

I'll think about non-latin character sets though, thanks for the input.

**Comment by Tusamarco** [comment]
Let me test it again ... but yes I had that issue.
If still happen I will let you know and open an issue as bug.. ok @akopytov ?

**Comment by akopytov** [comment]
@Tusamarco yes, please!
